---
 Makefile |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-CC       = g++
-ADDITIONAL_CFLAGS   = -g -O2 -march=i686 -Wall
+CXX       = g++
+ADDITIONAL_CFLAGS   = -g -O2 -Wall
 
 prefix   = /usr
 INCLUDES = -I$/usr/X11R6
@@ -28,14 +28,14 @@ OBJS     = windowmanager.o 	\
 all: aewm++
 
 aewm++: $(OBJS)
-	$(CC) $(OBJS) $(LDPATH) $(LIBS) -o $@
+	$(CXX) $(OBJS) $(LDPATH) $(LIBS) -o $@
 
 $(OBJS): %.o: %.cc $(HEADERS)
-	$(CC) $(CFLAGS) $(ADDITIONAL_CFLAGS) $(DEFINES) $(INCLUDES) -c $< -o $@
+	$(CXX) $(CFLAGS) $(ADDITIONAL_CFLAGS) $(DEFINES) $(INCLUDES) -c $< -o $@
 
 install: all
 	mkdir -p $(DESTDIR)$(prefix)/bin
-	mkdir -p $(DESTDIR)$(prefix)/man/man1
+	# mkdir -p $(DESTDIR)$(prefix)/man/man1
 	install -s aewm++ $(DESTDIR)$(prefix)/bin
 	
 clean:
