--- aewm++-1.1.2.orig/windowmanager.cc
+++ aewm++-1.1.2/windowmanager.cc
@@ -540,7 +540,7 @@
  	{
                 if( (unsigned)current_desktop != ks - XK_1 )
 		{
-			(unsigned)current_desktop = ks - XK_1;
+			current_desktop = ks - XK_1;
 			goToDesktop(current_desktop);
 		}
 	}
--- aewm++-1.1.2.orig/Makefile
+++ aewm++-1.1.2/Makefile
@@ -35,7 +35,6 @@
 
 install: all
 	mkdir -p $(DESTDIR)$(prefix)/bin
-	mkdir -p $(DESTDIR)$(prefix)/man/man1
 	install -s aewm++ $(DESTDIR)$(prefix)/bin
 	
 clean:
